src/components/topicCard/topicCard.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import "./topicStyle.scss";
+
+const topics = [
```

Comment:
this should be imported from the store to make sure things are functional later on

Reply:
+1 on this! This can be your next task to work on with @shannonbonet once ya'll get the `TopicView` finished and PRed so you can get everything merged together into a final workflow. 

You essentially would write another selector in the `TopicSlice` that you can select all of the relevant topics from the redux store, and then call that here to fetch your topics.